src/assets/hero.svg

```diff
@@ -0,0 +1,173 @@
+            <stop offset="1" stop-color="#89E4A3"/>
+        </linearGradient>
+        <linearGradient id="paint16_linear" x1="1033" y1="148" x2="1028.5" y2="248.5" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#F2B65C"/>
+            <stop offset="0.471634" stop-color="#FFDFAF"/>
+            <stop offset="1" stop-color="#FFE3B8"/>
+        </linearGradient>
+        <linearGradient id="paint17_linear" x1="1032.92" y1="254.125" x2="1032.92" y2="270.167" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#FFCBA6"/>
+            <stop offset="1" stop-color="#FFE5C7"/>
+        </linearGradient>
+        <linearGradient id="paint18_linear" x1="1025.65" y1="218.833" x2="1025.65" y2="284.695" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#ECA553"/>
+            <stop offset="1" stop-color="#EC8153"/>
+        </linearGradient>
+        <linearGradient id="paint19_linear" x1="1032.92" y1="183.542" x2="1032.92" y2="202.792" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#F9FCFF"/>
+            <stop offset="1" stop-color="#E5F1F9"/>
+        </linearGradient>
+        <linearGradient id="paint20_linear" x1="944.415" y1="199" x2="944.415" y2="267" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#B7E489"/>
+            <stop offset="1" stop-color="#89E4A3"/>
+        </linearGradient>
+        <linearGradient id="paint23_linear" x1="181.169" y1="456.858" x2="181.169" y2="655.196" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#9DC8E6"/>
+            <stop offset="1" stop-color="#C3E0F3"/>
+        </linearGradient>
+        <linearGradient id="paint24_linear" x1="115" y1="457" x2="31" y2="589" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#BBE4FF"/>
+            <stop offset="1" stop-color="#E5F1F9"/>
+        </linearGradient>
+        <linearGradient id="paint25_linear" x1="166.876" y1="358.825" x2="166.876" y2="494.048" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#FD6D6D"/>
+            <stop offset="1" stop-color="#DE5959"/>
+        </linearGradient>
+        <radialGradient id="paint21_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(258.308 349.4) rotate(104.501) scale(73.1257 88.7641)">
+            <stop stop-color="#FFD0A4"/>
+            <stop offset="1" stop-color="#F69F4E"/>
+        </radialGradient>
+        <radialGradient id="paint22_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(238.568 340.299) rotate(84.9825) scale(16.3418 70.4793)">
+            <stop stop-color="#FFBFA4"/>
+            <stop offset="1" stop-color="#F68A4E"/>
+        </radialGradient>
+        <clipPath id="clip0">
+            <rect x="955.915" y="129" width="154" height="154" fill="white"/>
+        </clipPath>
+        <clipPath id="clip1">
+            <rect y="286" width="290" height="394" fill="white"/>
+        </clipPath>
+    </defs>
+</svg>
```

Comment:
I don't like this... I think this will be be a nightmare to adapt to mobile, as it works for desktop cause the path is along the width, but if we collapse the grid, probably the path will be towards the bottom, along the height, and so this won't work anymore. Am I missing something?

Reply:
You have that problem with every possible approach. This can be just the desktop version and the mobile can use other svg or just use the things individually

Reply:
Ok let's roll with this then